topics/admin/tutorials/connect-to-compute-cluster/slides.html

```diff
@@ -0,0 +1,143 @@
+---
+layout: tutorial_slides
+topic_name: admin
+tutorial_name: connect-to-compute-cluster
+logo: "GTN"
+---
+layout: true
```

Comment:
Why is there a duplication in the metadata here?

Reply:
I understood why. I will revert my changes in other PR. But we should fix also the global layout if you think it should be like this